### PR TITLE
Toast Notification

### PR DESCRIPTION
**Pull request by manmodesanket**
## Description 
Toast notifications using vanilla js.

## Type
Machine Coding round.

## Related Technology
HTML, CSS, JavaScript.

## Source
I was asked this question in a interview. I was interviewing for Headout.

## Working Solution
Codesandbox link: https://codesandbox.io/s/toast-notification-h6cut

### CODE REVIEW

**Review by Jaynil1611 — CHANGES_REQUESTED** [review]
Please do the same as suggested in previous PR

**Comment by manmodesanket** [comment]
Made changes as per PR review

File: vanillaJs/toast-notification/src/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import "./styles.css";
+
+function addToast(data) {
+  let toastDiv = document.createElement("div");
```
Comment (Jaynil1611) [inline vanillaJs/toast-notification/src/index.js:4]: Change `let` to `const`

File: vanillaJs/toast-notification/src/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  toastDiv.classList.add(`toast__${data.type}`);
+  toastDiv.classList.add("toast");
+  toastDiv.innerHTML = data.content;
+  if (data.position === "true") {
+    console.log("called");
+    let toastDivError = document.querySelector(".toast__notifications__error");
```
Comment (Jaynil1611) [inline vanillaJs/toast-notification/src/index.js:10]: Change `let` to `const`

File: vanillaJs/toast-notification/src/index.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    toastDivError.appendChild(toastDiv);
+    toastDiv.classList.add(`toast__top`);
+    setTimeout(() => {
+      toastDivError.removeChild(toastDiv);
+    }, 2000);
+  } else {
+    let mainToastDiv = document.querySelector(".toast__notifications");
```
Comment (Jaynil1611) [inline vanillaJs/toast-notification/src/index.js:17]: Change `let` to `const`

File: vanillaJs/toast-notification/src/index.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    setTimeout(() => {
+      mainToastDiv.removeChild(toastDiv);
+    }, 2000);
+  }
+}
+
+function fun(e) {
+  let type = e.target.getAttribute("type");
+  let content = e.target.getAttribute("content");
+  let position = e.target.getAttribute("top");
```
Comment (Jaynil1611) [inline vanillaJs/toast-notification/src/index.js:28]: Change `let` to `const`

File: vanillaJs/toast-notification/src/styles.css [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  
+  /* common style for a toast notification*/
+  .toast {
+    padding: 0.5rem;
+    margin-bottom: 5px;
+    border-radius: 10px;
+  }
+  
+  .toast__info {
+    background-color: blue;
```
Comment (Jaynil1611) [inline vanillaJs/toast-notification/src/styles.css:30]: Change blue background to some light colour e.g. cyan

**Review by Jaynil1611 — CHANGES_REQUESTED** [review]

File: vanillaJs/toast-notification/README.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+## Toast Notification
+
+Q. Upon clicking Button1 and Button2 toast notification should popup at the bottom of the screen and if Button3 is clicked error notification should popup at the top of screen.
```
[the remark this answers is not in the document]
Comment (Jaynil1611) [reply vanillaJs/toast-notification/README.md:3]: Change button1 & button2, button3 as Info, Warning, Error in the problem description as well

**Review by Jaynil1611 — CHANGES_REQUESTED** [review]

**Comment by manmodesanket** [comment]
@Jaynil1611 I have updated button names in README.md. Please have a look.

**Review by Jaynil1611 — APPROVED** [review]

**Pull request merged**